components/list/list-en.html

```diff
@@ -32,3 +32,39 @@ <h3>Source code</h3>
 	&lt;li&gt;Item&lt;/li&gt;
 	&lt;li&gt;Item&lt;/li&gt;
 &lt;/ul&gt;</code></pre>
+
+<h2>Lists in columns <code>.list-col-[breakpoint]-[columns]</code></h2>
+<p>Add the class <code>list-col-[breakpoint]-[columns]</code> to display any list in a column format, where <code>[breakpoint]</code> can be either <code>xs</code>, <code>sm</code>, <code>md</code> or <code>lg</code> and <code>[columns]</code> can be either <code>1</code>, <code>2</code>, <code>3</code> or <code>4</code></p>
+<p>The smallest breakpoint defined will affect its breakpoint and all breakpoints above unless a larger breakpoint is defined. For example <code>list-col-sm-2</code> will display the list in two (2) columns for the <code>sm</code>, <code>md</code> and <code>lg</code> breakpoints. The default number of columns is one (1), so, in this example, the list would be displayed in one column for the <code>xs</code> breakpoint.</p>
+<p>Classes can be combined to display the items in a different amount of columns for different breakpoints. See following example.</p>
+<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3 list-col-lg-4">
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+	<li>Item</li>
+</ul>
+<h3>Source code</h3>
+<pre><code>
+&lt;ul class="list-col-xs-1 list-col-sm-2 list-col-md-3 list-col-lg-4"&gt;
```

Comment:
Just add some emphasis on the CSS class name

```suggestion
&lt;ul class="<strong>list-col-xs-1 list-col-sm-2 list-col-md-3 list-col-lg-4</strong>"&gt;
```

Reply:
> Just add some emphasis on the CSS class name

Done. :)